scaffolds/magic-stripe/scaffold.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Template, Zombi } from 'compiled/zombi';
+import { createScaffold } from 'cli/utils/scaffold-helpers';
+import { mergePrompts } from 'cli/utils/merge-prompts';
+import {
+  NpmClientPrompt,
+  PublishableApiKeyPrompt,
+  SecretApiKeyPrompt,
+  StripeSecretApiKeyPrompt,
+  StripePublishableApiKeyPrompt,
```

Comment:
Do we see a case where these prompts in particular are required across other templates? If no, I'd rather keep them in this scaffold file directly.

Reply:
If we have other scaffolds or options that used stripe, I'd say it should be shared. Otherwise, I think we could move it directly to the scaffold or similar.

Reply:
I think in this case it makes sense to avoid the unnecessary abstraction, but we can always revisit this decision in the future if we decide to add more "flavors" of Stripe + Magic.

Reply:
Good point. I moved them to a file within that specific scaffold. I like keeping them in their own file as it keeps the scaffold less noisy. Let me know if this is alright.

Reply:
Good with me!